pages/common/vegeta.md

```diff
@@ -0,0 +1,20 @@
+# vegeta
+
+> HTTP load testing tool.
+> Vegeta is a versatile HTTP load testing tool built out of a need to drill HTTP services with a constant request rate. It can be used both as a command line utility and a library.
+
+- Launch an attack during 30 seconds:
```

Comment:
Perhaps `lasting` would read better than `during` here?

Reply:
I choose to delete the first line. The second line is the official description from the vegeta repo and I prefer to take this one.